README.adoc

```diff
@@ -35,6 +37,39 @@ NOTE: If you want to use reveal.js 2, please switch your asciidoctor-reveal.js r
 
 TIP: If you are using https://pages.github.com/[GitHub Pages], plan ahead by keeping your source files on `master` branch and all output files on the `gh-pages` branch.
 
+[[node-package]]
+== Node package
+
+First you must install and configure {uri-nodejs-download}[Node.js] on your machine.
+
+Using npm:
+
+ $ npm i --save asciidoctor.js@1.5.5-2
+ $ npm i --save asciidoctor-template.js@1.5.5-2
+ $ npm i --save asciidoctor-reveal.js@1.5.5-2
+
+Once the package is installed, you can convert AsciiDoc files using Reveal.js backend.
+Here we are converting a file named `presentation.adoc` into a Reveal.js presentation using Node.js:
+
+.asciidoctor-revealjs.js
+[source, javascript]
+----
+// Load asciidoctor.js + asciidoctor-template.js
+var asciidoctor = require('asciidoctor.js')();
+var Asciidoctor = asciidoctor.Asciidoctor();
+var Opal = asciidoctor.Opal;
+Opal.load('nodejs');
+Opal.load('pathname');
+require('asciidoctor-template.js');
+
+// Convert the document 'presentation.adoc' using Reveal.js backend
+var attributes = 'revealjsdir=node_modules/asciidoctor-reveal.js/node_modules/reveal.js';
+var options = Opal.hash({safe: 'safe',
+                         backend: 'revealjs',
+                         attributes: attributes});
```

Comment:
Passing attributes as arguments makes overriding them in the document impossible. See [the documentation on attribute assignment precedence](http://asciidoctor.org/docs/user-manual/#attribute-assignment-precedence) for more information about that.

I suggest that you push the `revealjsdir` attribute default value in the example document as I suggested earlier.